radix-engine-interface/src/engine/wasm_input.rs

```diff
@@ -57,7 +57,7 @@ pub enum NativeFunctionInvocation {
 #[derive(Debug)]
 #[scrypto(TypeId, Encode, Decode)]
 pub enum ComponentMethodInvocation {
-    AddAccessCheck(ComponentAddAccessCheckInvocation),
+    AddAccessCheck(AccessRulesAddAccessCheckInvocation),
```

Comment:
Why do we still need `ComponentMethodInvocation`, given `add_access_check` is universal?

Reply:
Good catch